Source/JavaScriptCore/assembler/X86Assembler.h

```diff
@@ -5134,6 +5351,38 @@ class X86Assembler {
         m_formatter.vexNdsLigWigTwoByteOp(PRE_SSE_66, OP2_PMOVMSKB_EqWdq, (RegisterID)dest, (RegisterID)0, (RegisterID)input);
     }
 
+    void vmovaps_rr(XMMRegisterID input, XMMRegisterID dest)
+    {
+        // https://www.felixcloutier.com/x86/movaps
+        // VEX.128.0F.WIG 28 /r VMOVAPS xmm1, xmm2/m128
+        // A	NA	ModRM:reg (w)	ModRM:r/m (r)	NA	NA
```

Comment:
NANA => NA NA
And let's use 4 spaces for tabs un this line to align it to the other lines.